refinery/core/views.py

```diff
@@ -855,9 +856,46 @@ def update_group_perms(self, new_owner):
                 "groups_without_access": groups_without_access}
 
 
-class AnalysesViewSet(APIView):
-    """API endpoint that allows for Analyses to be deleted"""
-    http_method_names = ['delete']
+class AnalysisViewSet(APIView):
```

Comment:
Should this be named something like `AnalysisList` since it is derived from `APIView`?